package.json

```
@@ -12,6 +12,9 @@
   "devDependencies": {
     "@babel/core": "7.20.12",
     "@babel/preset-env": "7.20.2",
+    "@commitlint/cli": "^18.6.0",
+    "@commitlint/config-conventional": "^18.6.0",
+    "@commitlint/format": "^18.6.0",
```

use absolute version.. instead of ^
```suggestion
    "@commitlint/cli": "18.6.0",
    "@commitlint/config-conventional": "18.6.0",
    "@commitlint/format": "18.6.0",
```